README.md

```diff
@@ -15,6 +15,18 @@ file][config].
 
 [config]: http://eslint.org/docs/user-guide/configuring#using-configuration-files
 
+### Channels
+
+There are a few major versions of ESLint out there. The latest major version
+is 4. This engine provides a few channels for major versions of ESLint.
```

Comment:
will you make "channels" link to https://docs.codeclimate.com/docs/engine-channels please?